src/Command/BaseCommand.php

```diff
@@ -29,9 +29,28 @@
  * BaseCommand.
  *
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * NEXT_MAJOR: Remove this class, and for all commands that use this class need to extend from Symfony command.
+ *
+ * @deprecated since 3.27, and it will be removed in 4.0.
  */
 abstract class BaseCommand extends ContainerAwareCommand
 {
+    public function __construct(?string $name = null)
+    {
+        parent::__construct($name);
+
+        if (false === strpos($this->getName(), 'doctrine:')) {
```

Comment:
Sonata you meant ?

Reply:
Hey @VincentLanglet 

the commands from `custom_bootstrap` doesn't use the BaseCommand constructor, for this reason it was not working there!

I changed to `sonata:` then if found the command with sonata into the name the deprecation will show, But if someone extends this class and put other name that does't contain sonata it won't show :/

I thought to check by `doctrine:` but in theory any command different of `doctrine:` will show this deprecation, then I guess check with sonata is better :)

Reply:
Seems ok to me